### PR TITLE
Fix CombinedSchema loading/equality

### PR DESCRIPTION
**Pull request by tledkov**
Adds simple test case to illustrate unexpected and strange behavior.

### CODE REVIEW

**Comment by nizhikov** [comment]
```java
    @Test
    public void loadTheSameCombinedSeveralTimes() {
        JSONObject json = new JSONObject(new JSONTokener("{\"enum\": [\"V1\", \"V2\", \"V3\"],\"type\": \"string\"}"));

        for (int i = 0; i < Integer.MAX_VALUE; ++i) {
            Schema s0 = SchemaLoader.load(json);
            Schema s1 = SchemaLoader.load(json);

            System.out.println("Iter: " + i + ", equals=" + Objects.equals(s0, s1));

            if (i > 10) {
                assertEquals(s0, s1);
            }
        }
    }
    ```

**Comment by tledkov** [comment]
Hi there, 

I've pushed proposed fix for issue #497 and cleanup new tests.

**Comment by tledkov** [comment]
For fix take a look at the PR #498

File: core/src/test/java/org/everit/json/schema/loader/CombinedSchemaLoaderTest.java [hunk 1]
```diff
@@ -24,6 +25,7 @@
 public class CombinedSchemaLoaderTest {
 
     private static JSONObject ALL_SCHEMAS = ResourceLoader.DEFAULT.readObj("combinedtestschemas.json");
+    private static final int ITERS = 64;;
```
Comment (nizhikov) [inline core/src/test/java/org/everit/json/schema/loader/CombinedSchemaLoaderTest.java:28]: two commans in the end of the line.

**Pull request closed without merging**